release/pipelines/windows-customize/configmaps/windows-customize-configmaps.yaml

```diff
@@ -245,20 +245,36 @@ data:
           </AutoLogon>
           <FirstLogonCommands>
             <SynchronousCommand wcm:action="add">
-              <CommandLine>bcdedit /set device partition=C:</CommandLine>
+              <CommandLine>PowerShell -ExecutionPolicy Bypass -NoProfile F:\post-update.ps1</CommandLine>
```

Comment:
In general, I preferred to enumerate all drives for the post-update script, instead of hardcoding F:. F: means that you exactly know that Windows has only one drive C:, the second drive D: is a Windows ISO, etc...